### PR TITLE
Fix launcher configmap bug after spliting hadoop-service.

### PR DESCRIPTION
**Pull request by ydye**

### CODE REVIEW

File: pai-management/bootstrap/frameworklauncher/frameworklauncher.yaml.template [hunk 1]
```diff
@@ -65,9 +67,12 @@ spec:
       - name: launcher-log
         hostPath:
           path: {{ clusterinfo[ 'dataPath' ] }}/launcherlogs
-      - name: hadoop-config-volume
+      - name: hadoop-nm-config-volume
         configMap:
-          name: {{ clusterinfo[ 'hadoopinfo' ][ 'configmapname' ] }}
+          name: hadoop-node-manager-configuration
```
Comment (hwuu) [inline pai-management/bootstrap/frameworklauncher/frameworklauncher.yaml.template]: hadoop-nm-configuration?

File: pai-management/bootstrap/frameworklauncher/frameworklauncher.yaml.template [hunk 2]
```diff
@@ -65,9 +67,12 @@ spec:
...
+      - name: hadoop-dn-config-volume
+        configMap:
+          name: hadoop-data-node-configuration
```
Comment (hwuu) [inline pai-management/bootstrap/frameworklauncher/frameworklauncher.yaml.template]: hadoop-dn-configuration?

Comment (ydye) [reply pai-management/bootstrap/frameworklauncher/frameworklauncher.yaml.template]: Configmap name is hadoop-data-node-configuration. But the volume‘s name is hadoop-dn-config-volume.

Comment (hwuu) [reply pai-management/bootstrap/frameworklauncher/frameworklauncher.yaml.template]: Then why in generate-config.sh the cp command is cp  /hadoop-nm-configuration/core-site.xml ? Where is the folder /hadoop-nm-configuration defined?

Comment (ydye) [reply pai-management/bootstrap/frameworklauncher/frameworklauncher.yaml.template]: 3 Concept:
Configmap name： hadoop-node-manager-configuration
Volume Created by Configmap：hadoop-nm-config-volume
The path mounted by the volume：hadoop-nm-configuration

**Pull request merged**